packages/common/index.js

```diff
@@ -15,7 +15,7 @@ const browserSafe = {
   ...require("./src/SolidityTestUtils"),
   ...require("./src/TimeUtils"),
   ...require("./src/VotingUtils"),
-  ...require("./src/BlacklistedPriceRequests")
+  ...require("./src/PriceIdentifierUtils")
```

Comment:
This name made more sense: 
- We are dealing with Identifiers not Requests
- We can group all identifier-related mappings into this one file